website/public/js/controllers/menuCtrl.js

```diff
@@ -26,7 +26,7 @@ angular.module('habitrpg')
         }
       }
 
-      $scope.clearMessages = Chat.seenMessage;
+      $scope.clearMessages = Chat.markChatSeen;
       $scope.clearCards = Chat.clearCards;
```

Comment:
This will be undefined and throw an error anyone tries to call `$scope.clearCards` because `clearCards` wasn't exported. If this needs to be ported somewhere else, you should at least put a TODO comment here.